src/server/socketHandlers/scConnectionHandler.js

```diff
@@ -0,0 +1,21 @@
+import scSubscribeHandler from './scSubscribeHandler';
+import scUnsubscribeHandler from './scUnsubscribeHandler';
+import scGraphQLHandler from './scGraphQLHandler';
+
+export default function scConnectionHandler(exchange) {
+  return async function connectionHandler(socket) {
+    const subscribeHandler = scSubscribeHandler(exchange, socket);
```

Comment:
I'm not a fan of `.call, .apply`. Being explicit about _exactly_ what we want from `this` makes it easier to read IMO.